scripts/globals/effects/sigil.lua

```diff
@@ -16,6 +30,50 @@ end;
 -----------------------------------
 
 function onEffectTick(target,effect)
+    local inWotGzone = false;
+    local HPP = (target:getHP()/target:getMaxHP()) *100; -- Regen trigger is 45%-95% (% = subPower+10)
+    local MPP = (target:getMP()/target:getMaxMP()) *100; -- Refresh trigger is 35%-85% (% = subPower)
+    local tier = effect:getTier(); -- Tracks if regen/refresh already active or not.
+    local power = effect:getPower(); -- Tracks which bonus effects are in use.
+    local subPower = effect:getSubPower(); -- subPower sets % required to trigger regen/refresh.
+    -- Number of Campaign Area's controlled sets subPower at time of effect gain.
+    -- 10% diff between hp and mp triggers, so add 10 to subPower when comparing it to HP.
+
+    if (target:getCurrentRegion() >= 33 and target:getCurrentRegion() <= 40) then
+        inWotGzone = true;
+    end
+
+    if (inWotGzone == true) then -- Only works in WotG zones
+        if (HPP < subPower+10 and bit.band(tier, 1) == 0) then
+            if (power == 1 or power == 3 or power == 5 or power == 7 or power == 9 or power == 11 or power == 13 or power == 15) then
+                target:addMod(MOD_REGEN, 1);
+                effect:setTier(bit.bor(tier, (1)));
+            end
+        end
+
+        if (MPP < subPower and bit.band(tier, 2) == 0) then
+            if (power == 2 or power == 3 or power == 6 or power == 7 or power == 10 or power == 11 or power >= 14) then
+                target:addMod(MOD_REFRESH, 1);
+                effect:setTier(bit.bor(tier, (2)));
+            end
+        end
+    end
+
+    if ((inWotGzone == false or HPP >= subPower+10) and bit.band(tier, 1) > 0) then

```

Comment:
@teschnei can't for the life of me see where I goofed on setting the bits back to zero.